src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -110,7 +117,8 @@ internal class RecommendationServiceTest : ServiceTestBase() {
           lastModifiedDate = "2022-07-26T09:48:27.443Z",
           lastModifiedBy = "Bill",
           createdBy = existingRecommendation.data.createdBy,
-          createdDate = existingRecommendation.data.createdDate
+          createdDate = existingRecommendation.data.createdDate,
+          alternativesToRecallTried = existingRecommendation.data.alternativesToRecallTried
```

Comment:
Is it possible to add a test for the 'None' with details route to this test class? It doesn't end up in the Part A but we need to make sure it's saved in the database and retrieved in save and return route and I can't see any tests covering this.

Reply:
Done